HIPs/hedera-hashgraph.md

```diff
@@ -0,0 +1,304 @@
+
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the gRPC Web Proxy that today can be found at https://github.com/hashgraph/hedera-grpcWeb-proxy
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Exchange Rate Tool that today can be found at https://github.com/hashgraph/hedera-exchange-rate-tool
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the SDK TCK that today can be found at https://github.com/hashgraph/hedera-sdk-tck
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Mirror Node Explorer that today can be found at https://github.com/hashgraph/hedera-mirror-node-explorer
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Smart Contracts that today can be found at https://github.com/hashgraph/hedera-smart-contracts
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the JSON RPC Relay that today can be found at https://github.com/hashgraph/hedera-json-rpc-relay
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Local Node that today can be found at https://github.com/hashgraph/hedera-local-node
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Mirror Node that today can be found at https://github.com/hashgraph/hedera-mirror-node
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Services that today can be found at https://github.com/hashgraph/hedera-services
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the JavaScript SDK that today can be found at https://github.com/hashgraph/hedera-sdk-js
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Swift SDK that today can be found at https://github.com/hashgraph/hedera-sdk-swift
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Rust SDK that today can be found at https://github.com/hashgraph/hedera-sdk-rust
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Java SDK that today can be found at https://github.com/hashgraph/hedera-sdk-java
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Go SDK that today can be found at
+  https://github.com/hashgraph/hedera-sdk-go
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the C++ SDK that today can be found at https://github.com/hashgraph/hedera-sdk-cpp
+- `https://github.com/PROJECT/SUBPROJECT` - This repository contains the Solo CLI that today can be found at
+  https://github.com/hashgraph/solo
+
+As shown, the Hedera ecosystem already contains several sub-projects that must be part of `PROJECT`.
+All those sub-projects are necessary to deploy and run a full Hashgraph-based network and interact with that network.
+All mentioned projects are currently part of the Hashgraph organization at GitHub.
+Supplemental repositories include supporting tooling to deploy and validate a network built using the above codebases.
```

Comment:
Question:
Are these supplementary repositories currently part of the open-source hashgraph GitHub org? If yes, is there a plan to bring them as well to the LFDT?

Reply:
The hashgraph org currently contains way more repositories as we mentioned in this proposal. The mentioned repos here are the ones that we want to migrate "at the start". So yes, we plan to bring them to the LFDT. We will add more detail regarding the mentioned repos.